Move cli.Configdir away (in `flags`).

vdemeester opened the pull request:
This makes integration not depend anymore of `cli` and thus not
require `cobra` and other packages to compile.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

aaronlehmann commented on cli/flags/cli.go:
```diff
@@ -1,4 +1,4 @@
-package cli
+package flags
```
nit: Should we rename this file to `configdir.go` or something like that?

vdemeester replied:
Oh right thx 😇

aaronlehmann commented:
LGTM if CI passes

boaz0 reviewed (approved):
👍

vdemeester committed:
Move cli.Configdir away (in `flags`).

This makes integration not depend anymore of `cli` and thus not
require `cobra` and other packages to compile.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

thaJeztah reviewed (approved):
LGTM
